docs/configuration/nginx.md

```diff
@@ -155,8 +155,9 @@ Dokku uses a templating library by the name of [sigil](https://github.com/glider
 
 - Copy the following example template to a file named `nginx.conf.sigil` and either:
   - If using a buildpack application, you __must__ check it into the root of your app repo.
-  - If using Dockerfile deployment, set `WORKDIR` and `ADD` the file to this
-    directory (example: `WORKDIR /app` and `ADD nginx.conf.sigil /app`).
+  - If using a dockerfile or docker image for deploys, either:
+    - If WORKDIR is specified, add the file to the `WORKDIR` specified in the last Dockerfile stage  (example: `WORKDIR /app` and `ADD nginx.conf.sigil /app`).
+    - If no WORKDIR is specified, add the file to the root (`/`) of the docker image  (example:ADD nginx.conf.sigil /`).
```

Comment:
```suggestion
    - If no WORKDIR is specified, add the file to the root (`/`) of the docker image  (example: `ADD nginx.conf.sigil /`).
```

Reply:
Mind making a new pr for this change? I've already merged this PR...

Reply:
https://github.com/dokku/dokku/pull/4100